changelogs/fragments/letsencrypt.yml

```diff
@@ -0,0 +1,2 @@
+removed_features:
+- "The ``letsencrypt`` module has been removed. Use ``acme_certificate`` instead."
```

Comment:
Let's talk about this on #ansible-doc - we used to have these modules as links. @abadger - would it work if this was changed to R(acme_certificate_module) or something like that?

Reply:
This is for the changelog, it needs to use pure reStructuredText. When included in the docsite RST refs would work, but the changelog is (I think) often viewed as a text file on github (i.e. with basic RST formatting) - and then they won't work.